tests/core/test_serialization.py

```diff
@@ -38,6 +38,12 @@
     param baz = ({'a', 'b', 'c', 'd'}, frozenset({'e', 'f', 'g', 'h'}))
 """
 
+randomScenario = """
```

Comment:
If this scenario is only going to be used in one test I would move it into that test. If you want to keep it here I would at least give it a more informative name, since the `simpleScenario` above also has plenty of randomness in it.